qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -113,6 +113,9 @@ def main(self):
         self.add_options(parser)
         (self.options, self.args) = parser.parse_args()
 
+        if os.getenv("QT", ""):
+            self.options.noshutdown = True
```

Comment:
Any reason to do this? This will hinder executing `QT=1 ./qa/pull-tester/rpc-tests.py`?